viewer/vueapp/src/components/sessions/Sessions.vue

```diff
@@ -34,10 +34,11 @@
 
     <!-- visualizations -->
     <moloch-visualizations
-      v-if="mapData && graphData"
+      v-if="mapData && graphData && capStartTime"
```

Comment:
@awick The capture start time is only visible on the sessions page. Should this be added to all pages? I think it makes the most sense on the sessions page given the use case provided in #271

Reply:
ya i think anywhere there is sessions graph

Reply:
👍

Reply:
It is now displayed on all session timeline graphs! It is only shown on the main timeline graph in spigraph

<img width="1673" alt="Screen Shot 2020-10-16 at 8 23 33 AM" src="https://user-images.githubusercontent.com/1235082/96257788-dd172480-0f6f-11eb-8966-8c9158463783.png">